include/arbor/time_sequence.hpp

```diff
@@ -123,8 +123,8 @@ struct vector_time_seq {
         seq_(std::move(seq))
     {
         // Ensure that the time values are sorted.
-        if(!std::is_sorted(seq_.begin(), seq_.end())) {
-            util::sort(seq_);
+        if (!std::is_sorted(seq_.begin(), seq_.end())) {
+            std::sort(seq_.begin(), seq_.end());
```

Comment:
Is there a tipping point where the convenience of using the nice util functionality in the public API code outweighs the downsides of sharing it?

Reply:
Possibly, but I think we're fairly removed from that point.
(As you know, I don't think this guy should be calling sort anyhow!)